doc/SETUP.md

```diff
@@ -4,19 +4,22 @@
 
 - Install dependencies\
   `$ yarn install`
+- Create development .env file
+  `$ cp .env.example .env`
+- Set your desired .env settings to connect to either regtest or production for your specific browser.
 
 ### 💻 Load extension into browser
 
 - Start development build, which will automatically watch for file changes:
 
   - Chrome\
-    `$ yarn run dev:chrome`
+    `$ dotenv yarn run dev:chrome`
```

Comment:
Now we use dotenv does it make sense to also use cross-env? it could be removed and all env variables moved to the .env file